docs/src/misc.rst

```diff
@@ -291,6 +305,23 @@ API
 
     .. versionadded:: 1.9.0
 
+.. c:function:: int uv_get_passwd(uv_passwd_t* pwd)
+
+    Gets a subset of the password file entry for the current effective uid (not
+    the real uid). The populated data includes the username, uid, gid, shell,
+    and home directory. On non-Windows systems, all data comes from
+    `getpwuid_r()`. On Windows, uid and gid have no meaning, and shell is
```

Comment:
instead of "have no meaning" maybe also mention that they are set to -1 or something